### PR TITLE
Realclown

### PR DESCRIPTION
**Pull request by Eagle-Eyess**
:cl: EagleEyes
add: clowns cant abuse mutadone anymore
/:cl:

clowns were never meant to use guns (except for antags). you wanna use a gun dont play clown or get brain surgery... dummies

### CODE REVIEW

**Comment by Jujumatic** [comment]
Wait hold up this is going to make every single person who gets the clumsy trait completely unable to remove it

**Comment by Eagle-Eyess** [comment]
ill fix this later

**Comment by Jujumatic** [comment]
:)

**Review by yoyobatty — DISMISSED** [review]
Maybe create a sub mutation of clumsy called clown and replace the clowns mutation with it then add this var

**Comment by Manly-Dwarf** [comment]
yes

**Review by Jujumatic — DISMISSED** [review]
looks good though there might be a double indentation on line 6 or it's just my eyes fucking with me

**Comment by Eagle-Eyess** [comment]
this code should now finally work. (thanks yoyo)

File: hippiestation/code/datums/mutations/clumsy/clown.dm [hunk 1]
```diff
@@ -0,0 +1,5 @@
+/datum/outfit/job/clown/post_equip(mob/living/carbon/human/H)
+	..()
+	for(var/datum/mutation/human/M in H.dna.mutations)
+		if(istype(M, /datum/mutation/human/clumsy) && H.job == "Clown")
+			M.mutadone_proof = TRUE//jew jew fooking sucks and yoyo is better
```
Comment (Jujumatic) [inline hippiestation/code/datums/mutations/clumsy/clown.dm]: :japanese_goblin:

File: hippiestation.dme [hunk 2]
```diff
@@ -2870,6 +2870,7 @@
 #include "hippiestation\code\datums\martial\plasma_fist.dm"
 #include "hippiestation\code\datums\martial\sleeping_carp.dm"
 #include "hippiestation\code\datums\mood_events\generic_positive_events.dm"
+#include "hippiestation\code\datums\mutations\clumsy\clown.dm"
```
Comment (Jujumatic) [inline hippiestation.dme]: This isn't the path of the original proc we're editing though. You always make a modularised path which is based on the path of where you got the code from

**Review by Jujumatic — CHANGES_REQUESTED** [review]

**Comment by yoyobatty** [comment]
I also realized after that this is the job outfit equip therefore adding the job clown check is redundant

**Comment by Maroonenahmoron** [comment]
Yes, thank you! +1

**Review by Jujumatic — APPROVED** [review]

**Comment by Jujumatic** [comment]
this is unnecessarily expensive but fuck yoyo, fuck code and fuck the entire codebase being built on shitcode. also fuck tg cause why not

**Pull request merged**